src/v2/Apps/Conversation/Components/__tests__/Conversation.jest.tsx

```diff
@@ -2,13 +2,22 @@ import { ConversationPaginationContainer } from "../Conversation"
 import { graphql } from "react-relay"
 import { useTracking } from "react-tracking"
 import { useSystemContext } from "v2/System/useSystemContext"
-import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
+import { setupTestWrapperTL } from "v2/DevTools/setupTestWrapper"
 import { ConversationPagination_Test_Query } from "v2/__generated__/ConversationPagination_Test_Query.graphql"
+import { screen } from "@testing-library/react"
 
 jest.unmock("react-relay")
 jest.mock("v2/System/useSystemContext")
-
-const { getWrapper } = setupTestWrapper<ConversationPagination_Test_Query>({
+jest.mock("v2/Apps/Conversation/Components/ConfirmArtworkModal", () => ({
+  ConfirmArtworkModalQueryRenderer: () => null,
+}))
+jest.mock("v2/Apps/Conversation/Components/UnreadMessagesToast", () => ({
+  UnreadMessagesToastQueryRenderer: () => null,
+}))
```

Comment:
One thing we've been doing for testing existence-y components like this in RTL is to do 
```tsx
jest.mock("v2/Apps/Conversation/Components/UnreadMessagesToast", () => ({
  UnreadMessagesToastQueryRenderer: () => "UnreadMessagesToastQueryRenderer",
}))
```
which then mirrors our old enzyme tests in that we can do 
```tsx
expect(screen.queryByText("UnreadMessagesToastQueryRenderer")).toBeInDocument()
```
This may not be relevant here, but just a heads up for testing overall container-component composition.